src/utils/__tests__/calculateDatetimeDifference.test.js

```diff
@@ -20,4 +20,31 @@ describe('should calculate and return relative time diff between booking time an
       expect(formattedDateString).toEqual(expected);
     },
   );
+
+  it('should calculate and return relative time with suffix', () => {
+    const dateTimeArray = ['2020-10-24T01:15:00', '2020-11-08T14:00:00'];
+    const suffix = 'later';
+    const expected = '16 days later';
```

Comment:
So that _this_ is what it says:
```suggestion
    const expected = '28 days later';
```